src/components/accessories/tabs/Tabs.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from "classnames";
+import React, { Fragment, FunctionComponent, useState } from "react";
+import { IProps } from "./types";
+import "./styles.scss";
+
+const Tabs: FunctionComponent<IProps> = ({
+  header,
+  content,
+}) => {
+
+  const [ currentIndex, updateCurrentIndex ] = useState(0); 
+
+  const renderHeader = (item: object | any, index: number, mobile: boolean = false): JSX.Element => {
+    if(!mobile){
+      return (
+        <div className={classNames("tab", { active: (currentIndex === index) })} key={index} onClick={() => updateCurrentIndex(index)}>
+          <span>{item[index]}</span>
+        </div>
+      );
+    } else {
+      return (
+        <option className={classNames("tab", { active: (currentIndex === index) })} key={index} value={index}>
+          {item[index]}
+        </option>
+      );
+    }
+  };
+
+  const renderContent = (item: object | any, index: number): JSX.Element => {
+    return (
+      <div className="panel" key={index}>
+        <span>{item[index]}</span>
+      </div>
+    );
+  };
+
+  return (
+    <Fragment>
+      <div className={ header.mainClass }>
+        <div className="tabs desktop">
+          { header.items?.map((item, index): JSX.Element => renderHeader(item, index)) }
+        </div>
+        <div className="tabs mobile">
+          <select value={ currentIndex } onChange={(e) => updateCurrentIndex(parseInt(e.target.value)) }>
+            { header.items?.map((item, index): JSX.Element => renderHeader(item, index, true)) }
```

Comment:
I guess there's no need to declare the returning type here (I mean, the `JSX.Element`). It's already inferred by _Typescript_ (yes... I love it to).